src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  height: 80vmin;
+  width: 80vmin;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    bottom: 50%;
+    background-color: #2c8000;
+
+    transform-origin: bottom;
+    animation: rotate 60s linear infinite;
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    bottom: 50%;
+    background-color: #0700ff;
+
+    transform-origin: bottom;
+    animation: rotate 3600s steps(60) infinite;
```

Comment:
Similar to the previous comment, use a more descriptive name for the `@keyframes` animation to improve code readability and maintainability.